### ISSUE TITLE
aiohttp client support for Brotli compression

### ISSUE THREAD

**Issue by kakarukeys** [aio-libs/aiohttp#2270]
## Long story short

Currently unable to read brotli-compressed response. Is the support on the roadmap?

## Expected behaviour

Able to read.

## Actual behaviour

awaiting `response.text` will raise
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe7 in position 5: invalid continuation byte

## Steps to reproduce

```python
import asyncio, aiohttp
run = asyncio.get_event_loop().run_until_complete

async def f():
    async with aiohttp.ClientSession(headers={'Accept-Encoding': 'br'}) as session:
        async with session.get("https://www.amazon.com") as resp:
            html = await resp.text()

run(f())
```

## Your environment

Python 3.6.2
aiohttp==2.2.5
chardet==3.0.4
on Linux Mint 13

**Comment by asvetlov** [comment]
Never heard about Brotli before :)
But PR is welcome!

**Comment by fafhrd91** [comment]
I think brotli is from dropbox? should be easy to add.

yes, PR is always welcome

**Comment by asvetlov** [comment]
Fixed by #2312
